packages/wix-storybook-utils/src/typings/story-section.ts

```diff
@@ -22,27 +27,27 @@ export type Section =
   | LiveCodeSection
   | CodeSection
   | TabSection
-  | ApiSection;
+  | ApiSection
+  | ColumnsSection
+  | TableSection;
 
 export interface DescriptionSection extends StorySection {
-  text: string;
+  text: React.ReactNode | string;
```

Comment:
`string` is part of `React.ReactNode`, so 
```suggestion
  text: React.ReactNode;
```